Bullwinkle.class.nut

```diff
@@ -561,9 +595,7 @@ class Bullwinkle.Package {
             local d = date();
             return format("%d.%06d", d.time, d.usec);
         } else {
-            local d = math.abs(hardware.micros());
-            return format("%d.%06d", d/1000000, d%1000000);
+            return format("%d.%06d", time(), hardware.micros());  //this can be a bit of an ugly _ts but it allows us to calculate latencies up to 36 minutes long...
```

Comment:
Oh I see, `getLatency` only uses the seconds on the device. I missed that. Now, it's more accurate but even more ugly!